### ISSUE TITLE
Consider event sourcing projections

### ISSUE THREAD

**Issue by Swoorup** [edgedb/edgedb#1819]
This is both a question/request. One of the requirements of event sourcing application is projection/subscription to create read models.

Could edgedb provide mechanism to subscribe from a specific row in the events (append only table) and push changes to the consumers. That would eliminate the need for polling or message brokers like kafka.

**Comment by elprans** [comment]
We are planning to add support for event pub/sub via a configurable trigger system.

**Comment by b4stien** [comment]
Is this feature still planned? Is that the proper issue/discussion to track or is there another one?

**Comment by elprans** [comment]
Yes, EdgeDB 3.0 supports triggers, however a general pub/sub mechanism has not been implemented yet.  On the TODO list.
